scripts/hf_sim.py

```diff
@@ -301,8 +312,10 @@ def initialise(check_only=False):
                 dtype="f4",
             )
             # string parameters
-            if args.site_vm_dir != None:
-                vm = args.site_vm_dir
+            if args.site_v1d_dir != None:
+                vm = (
```

Comment:
Probably beyond scope but this variable `vm` refers to three different things depending on which function you are in. A bit of clarity around this would be awesome if possible.

Reply:
done